<details><summary>Linter check found the following problems:</summary>
Traceback (most recent call last):
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/anaconda_linter/run.py", line 208, in main
    return_code, report = execute_linter(
                          ^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/anaconda_linter/run.py", line 180, in execute_linter
    result = linter.lint(recipes, subdir, variant_config_files, exclusive_config_files, fix_flag)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/anaconda_linter/lint/__init__.py", line 725, in lint
    msgs = self.lint_file(
           ^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/anaconda_linter/lint/__init__.py", line 823, in lint_file
    recipe = _recipe.Recipe.from_file(
             ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/percy/render/recipe.py", line 243, in from_file
    raise exc from exc
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/percy/render/recipe.py", line 234, in from_file
    recipe._load_from_string(text.read())
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/percy/render/recipe.py", line 170, in _load_from_string
    self.render()
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/percy/render/recipe.py", line 290, in render
    self.meta = renderer_utils.render(self.recipe_dir, self.dump(), self.selector_dict, self.renderer)
                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/percy/render/_renderer.py", line 245, in render
    return ruamel.load(yaml_text.replace("\t", " ").replace("%", " "))
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/ruamel/yaml/main.py", line 451, in load
    return constructor.get_single_data()
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/ruamel/yaml/constructor.py", line 114, in get_single_data
    node = self.composer.get_single_node()
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/ruamel/yaml/composer.py", line 72, in get_single_node
    document = self.compose_document()
               ^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/ruamel/yaml/composer.py", line 94, in compose_document
    node = self.compose_node(None, None)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/ruamel/yaml/composer.py", line 130, in compose_node
    node = self.compose_mapping_node(anchor)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/ruamel/yaml/composer.py", line 211, in compose_mapping_node
    item_value = self.compose_node(node, item_key)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/ruamel/yaml/composer.py", line 130, in compose_node
    node = self.compose_mapping_node(anchor)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/ruamel/yaml/composer.py", line 204, in compose_mapping_node
    while not self.parser.check_event(MappingEndEvent):
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/ruamel/yaml/parser.py", line 141, in check_event
    self.current_event = self.state()
                         ^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/ruamel/yaml/parser.py", line 621, in parse_block_mapping_key
    self.move_token_comment(token)
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/ruamel/yaml/parser.py", line 815, in move_token_comment
    token.move_old_comment(self.scanner.peek_token() if nt is None else nt, empty=empty)
                           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/ruamel/yaml/scanner.py", line 1747, in peek_token
    self.fetch_more_tokens()
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/ruamel/yaml/scanner.py", line 217, in fetch_more_tokens
    self.stale_possible_simple_keys()
  File "/opt/conda/envs/linter/lib/python3.12/site-packages/ruamel/yaml/scanner.py", line 348, in stale_possible_simple_keys
    raise ScannerError(
ruamel.yaml.scanner.ScannerError: while scanning a simple key
  in "<unicode string>", line 47, column 1:
    {  set tests_to_skip = tests_to_ ...
    ^ (line: 47)
could not find expected ':'
  in "<unicode string>", line 48, column 1:
    {  set tests_to_skip = tests_to_ ...
    ^ (line: 48)

ERROR conda.cli.main_run:execute(125): `conda run conda-lint /tmp/abs_af8dos4j8j/clone` failed. (See above for error)
</details>

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -33,10 +33,10 @@ requirements:
 {% set tests_to_ignore = tests_to_ignore + " --ignore=tests/test_recursive_models.py" %}
 
 # OSError: Invalid argument
-{% set tests_to_skip = tests_to_skip + " or  test_handle_constrained_date[ge-le]%}  # [win]
-{% set tests_to_skip = tests_to_skip + " or  test_handle_constrained_date[gt-lt]%}  # [win]
-{% set tests_to_skip = tests_to_skip + " or  test_handle_constrained_date[ge-lt]%}  # [win]
-{% set tests_to_skip = tests_to_skip + " or  test_handle_constrained_date[gt-le]%}  # [win]
+{% set tests_to_skip = tests_to_skip + " test_handle_constrained_date[ge-le]%}     # [win]
+{% set tests_to_skip = tests_to_skip + " or test_handle_constrained_date[gt-lt]%}  # [win]
+{% set tests_to_skip = tests_to_skip + " or test_handle_constrained_date[ge-lt]%}  # [win]
+{% set tests_to_skip = tests_to_skip + " or test_handle_constrained_date[gt-le]%}  # [win]
 
 test:
   source_files:
@@ -45,7 +45,8 @@ test:
     - polyfactory
   commands:
     - pip check
-    - pytest -v tests {{ tests_to_ignore }} -k "not ({{ tests_to_skip }})"
+    - pytest -v tests {{ tests_to_ignore }} -k "not ({{ tests_to_skip }})"  # [win]
+    - pytest -v tests {{ tests_to_ignore }}                                 # [not win]
   requires:
     - pip
     - pytest

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -24,7 +24,7 @@ requirements:
     - faker
     - typing-extensions
 
-{% set tests_to_skip = "" %}
+{% set tests_to_skip = "" %} 
 {% set tests_to_ignore = "" %}
 # requires msgspec
 {% set tests_to_ignore = tests_to_ignore + " --ignore=tests/test_optional_model_field_inference.py" %}
@@ -33,10 +33,10 @@ requirements:
 {% set tests_to_ignore = tests_to_ignore + " --ignore=tests/test_recursive_models.py" %}
 
 # OSError: Invalid argument
-{% set tests_to_skip = tests_to_skip + " test_handle_constrained_date[ge-le]%}     # [win]
-{% set tests_to_skip = tests_to_skip + " or test_handle_constrained_date[gt-lt]%}  # [win]
-{% set tests_to_skip = tests_to_skip + " or test_handle_constrained_date[ge-lt]%}  # [win]
-{% set tests_to_skip = tests_to_skip + " or test_handle_constrained_date[gt-le]%}  # [win]
+{% set tests_to_skip = tests_to_skip + " test_handle_constrained_date[ge-le]"%}
+{% set tests_to_skip = tests_to_skip + " or test_handle_constrained_date[gt-lt]"%}
+{% set tests_to_skip = tests_to_skip + " or test_handle_constrained_date[ge-lt]"%}
+{% set tests_to_skip = tests_to_skip + " or test_handle_constrained_date[gt-le]"%}
 
 test:
   source_files:

--- a/recipe/meta.yaml
+++ b/recipe/meta.yaml
@@ -44,7 +44,7 @@ test:
   imports:
     - polyfactory
   commands:
-    - pip check
+    - pip check                                                             # [not osx]
     - pytest -v tests {{ tests_to_ignore }} -k "not ({{ tests_to_skip }})"  # [win]
     - pytest -v tests {{ tests_to_ignore }}                                 # [not win]
   requires:

